<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatherPad::PrefDialog</class>
 <widget class="QDialog" name="prefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="windowGoupBox">
     <property name="title">
      <string>Window</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="4">
       <widget class="QCheckBox" name="winSizeBox">
        <property name="text">
         <string>Remember window &amp;size on closing</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="sizeLable">
        <property name="toolTip">
         <string>Window frame is excluded.</string>
        </property>
        <property name="text">
         <string>Start with this size: </string>
        </property>
        <property name="indent">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinX">
        <property name="toolTip">
         <string>Window frame is excluded.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="mLabel">
        <property name="toolTip">
         <string>Window frame is excluded.</string>
        </property>
        <property name="text">
         <string>Ã—</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinY">
        <property name="toolTip">
         <string>Window frame is excluded.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QCheckBox" name="iconBox">
        <property name="toolTip">
         <string>Uncheck this if you want FeatherPad to
use system icons!

Needs application restart to take effect.</string>
        </property>
        <property name="text">
         <string>&amp;Use own icons</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QCheckBox" name="toolbarBox">
        <property name="text">
         <string>Do not show &amp;toolbar</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QCheckBox" name="menubarBox">
        <property name="toolTip">
         <string>If the menubar is hidden,
a menu button appears on the toolbar.</string>
        </property>
        <property name="text">
         <string>Do not show &amp;menubar</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QCheckBox" name="searchbarBox">
        <property name="text">
         <string>Hide search &amp;bar by default</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QCheckBox" name="statusBox">
        <property name="text">
         <string>Always show st&amp;atus bar</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="tabLabel">
        <property name="toolTip">
         <string>Will take effect after closing this dialog.</string>
        </property>
        <property name="text">
         <string>Tab position: </string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="3">
       <widget class="QComboBox" name="tabCombo">
        <property name="toolTip">
         <string>Will take effect after closing this dialog.</string>
        </property>
        <item>
         <property name="text">
          <string>North</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>South</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>West</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>East</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0" colspan="4">
       <widget class="QCheckBox" name="tabBox">
        <property name="toolTip">
         <string>This means that, for LTR, Alt+Right goes to the first tab
after the last tab is activated, and the same for Alt+Left.

Tab navigation with mouse wheel is not affected.</string>
        </property>
        <property name="text">
         <string>Tab navigation wraps &amp;around</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="4">
       <widget class="QCheckBox" name="singleTabBox">
        <property name="toolTip">
         <string>If this is checked, not only you will lose the informative
tooltip and context menu of a single tab but you could not
merge a single tabbed window into another one by tab drag-
and-drop either.</string>
        </property>
        <property name="text">
         <string>Do not show a single tab</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="textGoupBox">
     <property name="title">
      <string>Text</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="fontBox">
        <property name="toolTip">
         <string>Uncheck for Monospace 9.</string>
        </property>
        <property name="text">
         <string>Remember &amp;font</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="wrapBox">
        <property name="text">
         <string>&amp;Wrap lines by default</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="indentBox">
        <property name="text">
         <string>Auto-&amp;indent by default</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="lineBox">
        <property name="text">
         <string>Always show line &amp;numbers</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="syntaxBox">
        <property name="text">
         <string>Highlight s&amp;yntax by default</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="colBox">
        <property name="toolTip">
         <string>Needs file reopening to take effect.</string>
        </property>
        <property name="text">
         <string>Dark c&amp;olor scheme</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="colorValueLabel">
        <property name="toolTip">
         <string>The color value of the background.
255 means white while 0 means black.

For the light background, it can be
between 230 and 255; for the dark
background, between 0 and 50.

Needs file reopening to take effect.</string>
        </property>
        <property name="text">
         <string>Background color value: </string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="colorValueSpin">
        <property name="toolTip">
         <string>The color value of the background.
255 means white and 0 means black.

For the light background, it can be
between 230 and 255, and for the dark
background, between 0 and 50.

Needs file reopening to take effect.</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Never highlight syntax for files &gt; </string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="spinBox">
        <property name="suffix">
         <string> MiB</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QCheckBox" name="scrollBox">
        <property name="toolTip">
         <string>This is not a complete fix but
prevents annoying scroll jumps.</string>
        </property>
        <property name="text">
         <string>Workaround for &amp;Qt5's scroll jump bug</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset theme="edit-delete"/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="helpButton">
     <property name="text">
      <string>Help</string>
     </property>
     <property name="icon">
      <iconset theme="help-contextual"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
